skimage/filters/thresholding.py

```diff
@@ -366,3 +367,97 @@ def threshold_li(image):
             new_thresh = temp + tolerance
 
     return threshold + immin
+
+def threshold_minimum(image, nbins=256, bias='min') :
+    """ Return threshold value based on minimum method
+
+    Parameters
+    ----------
+    image : array
+        Input image.
+    nbins : int, optional
+        Number of bins used to calculate histogram. This value is ignored for
+        integer arrays.
+    bias : string, optional
+        'min', 'mid', 'max' return lowest, middle, or highest pixel value
+        with minimum histogram value
+
+    Returns
+    -------
+    threshold : float
+        Upper threshold value. All pixels intensities that less or equal of
```

Comment:
"Upper threshold value, i.e. all pixel intensities less than or equal to this threshold are foreground. May return 0 if the algorithm fails."